deploy/scripts/src/accounts/test.ts

```diff
@@ -174,10 +174,20 @@ export function changeEmail(): void {
     res = timeGroup(groups[1].split('::')[1], () => http.get(env.envURL, { redirects: 0 }), { isStatusCode302 })
 
     //02_OIDCStubCall
-    res = timeGroup(groups[2].split('::')[1], () => http.get(res.headers.Location), {
-      isStatusCode200,
-      ...pageContentCheck('API Simulation Tool')
-    })
+    res = timeGroup(
+      groups[2].split('::')[1],
+      () => {
+        const r = http.get(res.headers.Location)
+        if (!pageContentCheck('--API Simulation Tool--').validatePageContent(r)) {
```

Comment:
```suggestion
        if (!pageContentCheck('API Simulation Tool').validatePageContent(r)) {
```
I think these include `-` for debugging, remove these so they are the same as the assertion checks